WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/UsageDetails/CouponRestrictionsViewModel.swift

```diff
@@ -113,3 +137,26 @@ final class CouponRestrictionsViewModel: ObservableObject {
         self.storageManager = storageManager
     }
 }
+
+private extension CouponRestrictionsViewModel {
+    enum Localization {
+        static let excludeProducts = NSLocalizedString(
+            "Exclude Products",
+            comment: "Title of the action button to add products to the exclusion list in Coupon Usage Details screen"
+        )
+        static let excludedProducts = NSLocalizedString(
+            "Exclude Products (%1$d)",
+            comment: "Title of the action button to add excluded products on Coupon Usage Details screen. " +
+            "The number of selected items is included between the brackets. Reads like: Exclude Products (2)"
+        )
+        static let excludeProductCategories = NSLocalizedString(
+            "Exclude Product Categories",
+            comment: "Title of the action button to add product categories on Coupon Usage Details screen"
+        )
+        static let excludedProductCategories = NSLocalizedString(
+            "Exclude Product Categories (%1$d)",
+            comment: "Title of the action button to add excluded product categories on Coupon Usage Details screen. " +
+            "The number of selected items is included between the brackets. Reads like: Exclude Product Categories (4)"
```

Comment:
super nit: `The number of selected items is included between the brackets.` I would probably replace the word `included` with `mentioned`, just because `included` and `excluded` have specific meanings in this context. 😅